src/main/java/com/sforce/async/BulkConnection.java

```diff
@@ -82,6 +82,9 @@ public class BulkConnection {
     public static final String ZIP_XML_CONTENT_TYPE = "zip/xml";
     public static final String ZIP_CSV_CONTENT_TYPE = "zip/csv";
     public static final String ZIP_JSON_CONTENT_TYPE = "zip/json";
+    public static final String AVTO_BINARY_CONTENT_TYPE = "avro/binary";
+    public static final String OCTET_STREAM_CONTENT_TYPE = "application/octet-stream";
```

Comment:
I think we can use application/vnd.apache.parquet for parquet....
https://www.iana.org/assignments/media-types/application/vnd.apache.parquet

Reply:
done